src/auto/color.rs

```diff
@@ -31,6 +31,18 @@ impl Color {
     fn to_string(&self) -> GString {
         unsafe { from_glib_full(pango_sys::pango_color_to_string(self.to_glib_none().0)) }
     }
+
+    pub fn red(&self) -> u16 {
```

Comment:
This should probably go into a new file outside of auto crate, right?

Reply:
Yeah, otherwise it will get lost if this file gets regenerated